.github/workflows/docker-image.yml

```
@@ -56,6 +56,7 @@ jobs:
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
   deploy:
+    permissions: none
```

actionlint (1.7.4)

"none" is invalid for permission for all the scopes. available values are "read-all" and "write-all"

(permissions)